README.md

```diff
@@ -35,7 +35,13 @@ The Model Card Toolkit is hosted on [PyPI](https://pypi.org/project/model-card-t
 
 ## Model Card Generation on TFX
 
-If you are using [TensorFlow Extended (TFX)](https://www.tensorflow.org/tfx), you can incorporate model card generation into your TFX pipeline via the [`ModelCardGenerator`](https://www.tensorflow.org/responsible_ai/model_card_toolkit/api_docs/python/model_card_toolkit/ModelCardGenerator) component. See our [guide](model_card_toolkit/documentation/guide/templates.md) for more details, [this case study](model_card_toolkit/documentation/examples/MLMD_Model_Card_Toolkit_Demo.ipynb) for a demonstration.
+If you are using [TensorFlow Extended (TFX)](https://www.tensorflow.org/tfx), you can
+incorporate model card generation into your TFX pipeline via the `ModelCardGenerator`
+component. The `ModelCardGenerator` component is moving to the
+[TFX Addons](https://github.com/tensorflow/tfx-addons) library and will not be
+included in Model Card Toolkit from version 1.4.0. This page will be updated to
```

Comment:
we may want to consider doing a major release for this breaking change. Not 100% sure though.

Reply:
tl;dr: same, tbh

Definitely worth considering! I'd looked at [Python's Backwards Compatibility Policy](https://peps.python.org/pep-0387) for guidance, and a 1.4.0 release for sure goes against their policies of not removing a feature without notice between two consecutive releases and of giving warnings in at least two minor versions. On the other hand, this change removes a major blocker and your idea to use an optional import is a great way mitigate the breakage it introduces. 

There was [previous discussion](https://github.com/tensorflow/model-card-toolkit/issues/186#issuecomment-1031944358) of a 2.0 release moving away from enforcing the proto based schema and using Lit instead of jinja for rendering the model card. A major release makes sense, but I wonder about the timing if we consider these previously planned/promised changes.

Reply:
As decided on Slack, we're moving forward with a 2.0.0 release and can introduce other features in later versions.